backend/pkg/transport/network/tcp/tcp_test.go

```diff
@@ -0,0 +1,86 @@
+package tcp_test
+
+import (
+	"encoding/json"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/transport/network/tcp"
+	"github.com/rs/zerolog"
+	"os"
+	"testing"
+	"time"
+)
+
+type Socket struct {
+	SrcIP   string
+	SrcPort uint16
+	DstIP   string
+	DstPort uint16
+}
+
+// Payload defines a piece of information comming from the network with its metadata
+type Payload struct {
+	Timestamp time.Time
+	Socket    Socket
+	Data      []byte
+}
+
+func TestTCP(t *testing.T) {
+	writer := os.Stdout
+	logger := zerolog.New(writer)
+
+	// Server setup
+	serverAddr := "127.0.0.1:3000"
+	server := tcp.NewServer(serverAddr, tcp.ServerConfig{}, logger)
```

Comment:
instead of creating an empty config, create a method in the file `tcp/config.go` named `NewServerConfig` that creates a new `ServerConfig` with the following defaults:
```go
tcp.ServerConfig{
	ListenConfig: net.ListenConfig{
		KeepAlive: time.Second,
	},
	Context: context.TODO(),
}
```
These defaults are the ones being currently used, but feel free to propose some better defaults